src/components/reusable/timepicker/timepicker.stories.js

```diff
@@ -1,76 +1,91 @@
 import { html } from 'lit';
 import './index';
 import { action } from '@storybook/addon-actions';
-import { ifDefined } from 'lit/directives/if-defined.js';
-import '@kyndryl-design-system/shidoka-foundation/components/icon';
+import { useEffect } from '@storybook/addons';
 
 export default {
   title: 'Components/Timepicker',
   component: 'kyn-time-picker',
-  argTypes: {
-    size: {
-      options: ['sm', 'md', 'lg'],
-      control: { type: 'select' },
-    },
-    minTime: {
-      control: { type: 'text' },
-    },
-    maxTime: {
-      control: { type: 'text' },
-    },
-    step: {
-      control: { type: 'text' },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/s9VKYHFn1GncFyxd5l19nU/1.11-Amsterdam?node-id=6086-1559&node-type=canvas&m=dev',
     },
   },
+  argTypes: {
+    locale: { control: { type: 'text' } },
+    minTime: { control: { type: 'text' } },
+    maxTime: { control: { type: 'text' } },
+    defaultDate: { control: { type: 'text' } },
+    twentyFourHourFormat: { control: { type: 'boolean' } },
+  },
 };
 
-const args = {
-  unnamed: 'Time',
-  size: 'md',
-  name: 'timepicker',
-  value: '',
-  caption: '',
-  required: false,
-  disabled: false,
-  invalidText: '',
+const disconnectFlatpickr = () => {
+  const calendarElements = document.querySelectorAll('.flatpickr-calendar');
+  calendarElements.forEach((calendar) => calendar.remove());
+};
+
+const Template = (args) => {
+  useEffect(() => {
+    return () => {
+      disconnectFlatpickr();
+    };
+  }, []);
+
+  return html`
+    <kyn-time-picker
+      .nameAttr="${args.nameAttr}"
+      .locale="${args.locale}"
+      ?required="${args.required}"
+      .value="${args.value}"
```

Comment:
When I set 12:00 AM on the timepicker, it appears as `00:00`

Reply:
this is a flatpickr thing, i think. you can test here:

https://flatpickr.js.org/examples/#time-picker

Reply:
I see, interesting default. I wonder if we can [change the default format](https://flatpickr.js.org/formatting/#time-formatting-tokens) so it shows as "12:00 AM"? Actually, I don't see a prop on the timepicker component to modify the format like datepicker has.

Reply:
from what i could tell from the docs, the dateFormat doesn't behave quite the same for Timepicker as it does for Datepicker. i tried another time format like `H:i:s` and it didn't work.

that's why i'm hardcoding that value, but i can try again to make it work and get the 12H format to show in the input

Reply:
I wonder if something like `h:i K` is what we want there for the 12 hour format default.

Reply:
that works, nice 👍

Reply:
✅